docs/extensibility/custom-resources.md

```diff
@@ -106,7 +106,7 @@ Press <kbd>Ctrl</kbd>+<kbd>C</kbd> to shutdown the app (you can close the browse
 1. Add `Aspire.Hosting` to the class library as a package reference.
 
     ```dotnetcli
-    dotnet add ./MailDev.Hosting\MailDev.Hosting.csproj package Aspire.Hosting --version 8.0.0
+    dotnet add ./MailDev.Hosting/MailDev.Hosting.csproj package Aspire.Hosting --version 8.0.0-preview.7.24251.11
```

Comment:
Would it make sense to also make a special note/remark somewhere to say that you'd want this version to match the version of Aspire.Hosting used in the AppHost project? Just mentioning it since  the way that the person from the video figured it out was just by using the workload version, so that ended up being the way they fixed it. This comment would help in that, even if the docs are outdated, the user reading them can do the right thing.